OneLogin/functions/Set-OneLoginUserPasswd.ps1

```diff
@@ -0,0 +1,43 @@
+function Set-OneLoginUserPasswd
+{
+    [CmdletBinding(SupportsShouldProcess, ConfirmImpact = "Medium")]
+    param
+    (
+        [Parameter(Mandatory = $true, ValueFromPipeline)]
+        [ValidateNotNullOrEmpty()]
+        [OneLogin.User]
+        $Identity,
+
+        [Parameter(Mandatory = $true)]
+        [ValidateNotNullOrEmpty()]
+        [string]

```

Comment:
Password parameters should be of type `[SecureString]` as a best practice. That may make comparison of values a bit more difficult but should be considered.

Reply:
I thought about this and the way I implemented it in my automation, I decrypted the secure string before calling the function. Overall, I think it would be better to require the user to provide a secure string and facilitate the decryption within the function.

I can look into updating and testing that.